docs/doxygen-user/installActiveMQ.dox

```diff
@@ -1,61 +1,80 @@
 <br><br>
 \image html wherejava.PNG
 <br><br>   
-You can test if you have a 32-bit or 64-bit JRE installed via the following:
-    - _java -d32 -version_ &nbsp;&nbsp;&nbsp; for a 32-bit JRE
-    - _java -d64 -version_ &nbsp;&nbsp;&nbsp; for a 64-bit JRE
-<br><br>
-The screenshot below shows that there is a no 32-bit JRE on this machine, and there is a 64-bit JRE.
-<br><br>
-\image html JRE_bitness.PNG
+If you need the JRE, install it with the default settings.
+
+
+\subsection install_activemq_install_mq ActiveMQ Installation
+
+1. Extract the contents of the ActiveMQ archive folder to a location of your choice, bearing in mind that the files should be in a location that the running process will have write permissions to the folder. A typical folder choice is <i>C:\\Program Files\\apache-activemq-5.11.1</i>. Typically, it will ask for administrator permission to move the folder. Allow it if required. 
+
+2.	Edit the <i>conf\\activemq.xml</i> in the extracted folder to add <i>"&amp;wireFormat.maxInactivityDuration=0"</i> to the URI for the _transportConnector_ named _openwire_. Add the text highlighted in yellow below:
 <br><br>
-If you do not have a JRE installed, proceed to step 2. If you have a JRE installed, proceed to step 3.
+\image html maxinactivityduration.PNG
 <br><br>
-2.	Install the appropriate 32 or 64-bit version of the JRE, depending upon the version of Autopsy you have installed. Download one from: http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html. If you installed 32-bit Autopsy, be sure to select a package that has "x86" in the name. If you installed 64-bit Autopsy, be sure to select a package that has "x64" in the name. Follow the installation prompts to install the JRE.
-3.	Download ActiveMQ-5.11.1 from: http://activemq.apache.org/activemq-5111-release.html
-4.	Extract the files in the archive
-5.	Edit <i>apache-activemq-5.11.1\\conf\\activemq.xml</i> to add <i>"&amp;wireFormat.maxInactivityDuration=0"</i> to the URI for the _transportConnector_ named _openwire_. Add the text highlighted in yellow below:
+
+
+3.	Install ActiveMQ as a service by navigating to the folder <i>bin\\win64</i>, right-clicking _InstallService.bat_, clicking _Run as administrator_, then click _Yes_.
+
+4.  Start the ActiveMQ service by pressing _Start_, type _services.msc_, and press _Enter_. Find _ActiveMQ_ in the list and press the _Start the service_ link.
+
+5. ActiveMQ should now be fully installed and configured. You can access the admin pages in your web browser via a URL like this (set your host): http://localhost:8161/admin. The default administrator username is _admin_ with a password of _admin_ and the default regular username is _user_ with a default password of _password_.  You can change these passwords by following the instructions below. If you can see a page that looks like the following, it is ready to function.
```

Comment:
I don't call it fully configured by this point. They really should add some authentication setup. Or we should at least say that it's installed, but optional configuration can be made.